Source/WebCore/editing/MarkupAccumulator.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2004-2017 Apple Inc. All rights reserved.
+ * Copyright (C) 2004-2022 Apple Inc. All rights reserved.
  * Copyright (C) 2009, 2010 Google Inc. All rights reserved.
```

Comment:
Given this change is inspired by a change in chromium made by a Googler, we should probably attribute the copyright to Google as well.

Reply:
Done - updated Google to 2009-2022 and reverted Apple back to original.